swan-lake/learn-the-platform/source-code-dependencies/manage-dependencies.md

```diff
@@ -122,18 +122,18 @@ The local repository is useful to test a package in the development phase or to
 
 1. Generate the Ballerina archive after editing the package source files as required.
 
-   ```bash
+   ```
    bal pack
    ```
 
 2. Publish to the local repository.
-   ```bash
+   ```
    bal push --repository local
    ```
 
    If you already have the path of Ballerina archive, then you can simply execute the following command.
 
-    ```bash
+    ```
     bal push --repository local <path-to-bala-archive>
     ```
```

Comment:
Any particular reason remove the `bash` tag?

Reply:
IIANM, bash is not supported in the code snippets of the new site. @sm1990 - Please confirm.

Reply:
@NipunaRanasinghe & @praneesha as the original bug shows, shiki seems to be highlighting `bash` keywords even though they're not inside a bash script. Therefore we decided to remove the `bash` tag so that they will render without the highlighting.

Reply:
@sm1990 I see.. However we might still need some sort of highlighting(special rendering) even if the code block does not have any language tag. If we have fixed that issue, +1 to remove the bash keyword.

Reply:
The code blocks are rendered fine now after removing `bash` as shown below.

<img width="1679" alt="Screenshot 2022-08-09 at 11 25 18" src="https://user-images.githubusercontent.com/11707273/183575199-d6dcdac9-45b8-4fa6-8d20-975a35191a79.png">

<img width="1678" alt="Screenshot 2022-08-09 at 11 25 04" src="https://user-images.githubusercontent.com/11707273/183575220-ddc7b4d3-552c-4e6f-ac9c-e01980b75776.png">